src/seqan2seqan3.hpp

```diff
@@ -362,7 +380,7 @@ struct seqan2_to_rank_inner
     template <typename t>
     constexpr uint8_t operator()(t const c) const noexcept
     {
-        if constexpr (std::is_same_v<t, seqan3::aa27> /* or dna3bs*/)
+        if constexpr (std::is_same_v<t, seqan3::aa27> || std::is_same_v<t, seqan3::dna5>)
```

Comment:
```suggestion
        if constexpr (std::is_same_v<t, seqan3::aa27> || std::is_same_v<t, seqan3::dna5>) // dna5 used in bisulphite-mode
```

Reply:
This is unnecessary for regular dna-alignment, but all "fixes" would be quite complicated, so let's leave it like this.